app/services/invite_schools.rb

```diff
@@ -273,6 +273,28 @@ def invite_sitless_not_opted_out_schools_for_nqt_plus_one
     end
   end
 
+  def invite_not_opted_out_sits_with_all_validated_participants_for_nqt_plus_one
+    School
+      .eligible
+      .not_opted_out
+      .joins(induction_coordinators: :induction_coordinator_profile)
+      .all_ecf_participants_validated
+      .find_each do |school|
+        sit = school.induction_coordinators.first
+
+        next if sit.induction_coordinator_profile.year2020_invite_sent_at.present?
```

Comment:
We have a thing that does this automatically for mailers now :) Check out https://github.com/DFE-Digital/early-careers-framework/pull/1158/files#diff-94e26069880f723002215e411bfcbc43226733d19776c741a84f7fb9d57d30eeR281 and https://github.com/DFE-Digital/early-careers-framework/pull/1158/files#diff-4c570b8127fbf99f726f6b1a409d563cc29b16e9c464c92b90173c71b58ef416R152 for usage.

Reply:
oooh fancy

Reply:
replaced this now